core/silkworm/state/block_state.hpp

```diff
@@ -0,0 +1,40 @@
+/*
+   Copyright 2020-2021 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#ifndef SILKWORM_HEADER_ONLY_CHAIN_STATE_HPP_
```

Comment:
Should be `SILKWORM_STATE_BLOCK_STATE_HPP_` (header guards should be `<PATH>_<FILE_NAME>_HPP_`)

Reply:
ok!